### PR TITLE
APP: Move plugins to configurable folder

### PR DESCRIPTION
**Pull request by oslabbie**

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -38,7 +38,13 @@ Plugins can be used to extend the functionality of Luminary.
 
 #### How to add plugins
 
--   To add a plugin you need to add the new typescript file in the [plugins folder](./src/plugins/)
+-   To add a plugin you need to create a plugins folder, somewhere out of the project structure
+-   Set the VITE_PLUGIN_PATH env variable in your .env, then vite will go fetch your files at that location and copy it into the [plugins folder](./src/plugins/)
```
Comment (ivanslabbert) [inline app/README.md:42]:
```suggestion
-   Set the `VITE_PLUGIN_PATH` env variable in your .env for vite to copy your plugins to the [plugins folder](./src/plugins/)
```

File: app/README.md [hunk 2]
```diff
@@ -38,7 +38,13 @@ Plugins can be used to extend the functionality of Luminary.
...
+
+```
+VITE_PLUGIN_PATH="../../plugins"
+```
+
 -   To let luminary run the plugin you need to add a env variable VITE_PLUGINS to your env file and add an array of the plugins you want to add
```
Comment (ivanslabbert) [inline app/README.md:48]:
```suggestion
-   To let luminary run the plugin you need to add an env variable `VITE_PLUGINS` to your env file with an array of plugin names.
```

File: app/README.md [hunk 3]
```diff
@@ -47,6 +53,8 @@ VITE_PLUGINS=["examplePlugin", "examplePlugin2"]
 
 -   Every plugin class should have a constructor function
 
+**The files is being copied everytime before vite build, dev, and test is run**
```
Comment (ivanslabbert) [inline app/README.md:56]:
```suggestion
**The files are copied before vite build, dev, and test runs**
```

File: app/README.md [hunk 4]
```diff
@@ -38,7 +38,13 @@ Plugins can be used to extend the functionality of Luminary.
 
 #### How to add plugins
```
Comment (ivanslabbert) [inline app/README.md:39]:
```suggestion
#### Adding plugins
```

**Review by ivanslabbert — CHANGES_REQUESTED** [review]
Can you explain a bit how plugins can access the vue app? I see it is exported now, so I guess it will be through that export?

**Comment by oslabbie** [comment]
Hey @ivanslabbert 

Yes it is through the export, 
If you know of a better way, welcome to suggest it, then i can impliment it

**Review by ivanslabbert — APPROVED** [review]

**Pull request merged**